packages/alice_objectbox/lib/model/cached_alice_http_request.dart

```diff
@@ -0,0 +1,124 @@
+import 'dart:convert' show jsonDecode, jsonEncode;
+import 'dart:io' show Cookie;
+import 'package:alice/model/alice_form_data_file.dart';
+import 'package:alice/model/alice_from_data_field.dart';
+import 'package:alice/model/alice_http_request.dart';
+import 'package:alice_objectbox/json_converter/alice_form_data_field_converter.dart';
+import 'package:alice_objectbox/json_converter/alice_form_data_file_converter.dart';
+import 'package:objectbox/objectbox.dart';
+
+@Entity()
+class CachedAliceHttpRequest implements AliceHttpRequest {
+  CachedAliceHttpRequest({
+    this.objectId = 0,
+    this.size = 0,
+    DateTime? time,
+    this.headers = const <String, dynamic>{},
+    this.body = '',
+    this.contentType = '',
+    this.cookies = const [],
+    this.queryParameters = const <String, dynamic>{},
+    this.formDataFiles,
+    this.formDataFields,
+  }) : time = time ?? DateTime.now();
+
+  @Id()
+  int objectId;
```

Comment:
Shouldn't we have final fields here?

Reply:
https://github.com/jhomlala/alice/pull/205#discussion_r1658285361